src/groups/special_euclidean.jl

```diff
@@ -128,7 +144,11 @@ The formula for the coefficients reads ``t×(R⋅ω) + R⋅r`` for the translati
 matrix part of `p`, `r` is the translation part of `fX` and `ω` is the rotation part of `fX`,
 ``×`` is the cross product and ``⋅`` is the matrix product.
 """
-function adjoint_action(::SpecialEuclidean{3}, p, fX::TFVector{<:Any,VeeOrthogonalBasis{ℝ}})
```

Comment:
Would this still work for the field-manifold?

Reply:
No, it wouldn't, but I think we concluded that it's fine if type-parameter is the default (which it is now).

Reply:
Ok, cool.
For me this is a good tradeoff, now that I thought about that for a while.

It is like, for the day-to-day work you have a, maybe not that efficient, but reliable and tested car (parameter type), that does the job, but is maybe not fast in first runs.
And if you have to race, then you switch to the race car (field type) that is fast also in first runs, but (not yet) so reliable and maybe without airbags.

Reply:
Cool, I also think it makes sense.